tests/migration/install_all_patterns.pm

```diff
@@ -0,0 +1,21 @@
+# SLE15 migration tests
+#
+# Copyright 2023 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Install patterns for allpatterns cases before conducting migration
+# Maintainer: QE YaST and Migration (QE Yam) <qe-yam at suse de>
+
+use base "consoletest";
+use strict;
+use warnings;
+use testapi;
```

Comment:
Please just include functions used here.

Reply:
Could you explain me better what we need to include? At the moment it is included only the used modules

Reply:
In your test module, only used select_console, so include the function of select_console is enough.